ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/parser/SerialTreeParser.java

```diff
@@ -123,7 +123,7 @@ public Collection<Node> parseDeclarations(String source) throws TreeParserExcept
     private boolean isModuleDeclarationAllowed(ModuleMemberDeclarationNode declarationNode) {
         if (declarationNode instanceof FunctionDefinitionNode) {
             String functionName = ((FunctionDefinitionNode) declarationNode).functionName().text();
-            if (RESTRICTED_FUNCTION_NAMES.contains(functionName)) {
```

Comment:
Sorry, I missed this before :(  We can remove the unsued `RESTRICTED_FUNCTION_NAMES` variables from `SerialTreeParser` and `ModuleMemberTrail` classes.